jd-geek-source/src/app/models/product.interface.ts

```diff
@@ -0,0 +1,180 @@
+import { CategoryPath } from './categoryPath.interface'
+import { Images } from './images.interface'
+import { IncludedItemList } from './includedItemList.interface'
+import { Shipping } from './shipping.interface'
+import { ShippingLevelsOfService } from './shippingLevelsOfService.interface'
+
+export interface Product {
+  sku: number,
+  score: string,
+  productId: string,
+  name: string,
+  source: string,
+  type: string,
+  startDate: string,
+  new: boolean,
+  active: boolean,
+  lowPriceGuarantee: boolean,
+  activeUpdateDate: string,
+  regularPrice: number,
+  salePrice: number,
+  clearance: boolean,
+  onSale: boolean,
+  planPrice: string,
+  priceWithPlan: [],
+  contracts: [],
+  priceRestriction: string,
+  priceUpdateDate: string,
+  digital: boolean,
+  preowned: boolean,
+  carriers: [],
+  planFeatures: [],
+  devices: [],
+  carrierPlans: [],
+  technologyCode: string,
+  carrierModelNumber: string,
+  earlyTerminationFees: [],
+  monthlyRecurringCharge: number,
+  monthlyRecurringChargeGrandTotal: number,
+  activationCharge: number,
+  minutePrice: number,
+  planCategory: string,
+  planType: string,
+  familyIndividualCode: string,
+  validFrom: string,
+  validUntil: string,
+  carrierPlan: string,
+  outletCenter: string,
+  secondaryMarket: string,
+  frequentlyPurchasedWith: [],
+  accessories: [],
```

Comment:
an array of what?

Reply:
No idea.  That is how it come across from BB